metals/src/main/scala/scala/meta/internal/metals/PackageProvider.scala

```diff
@@ -1,40 +1,55 @@
 package scala.meta.internal.metals
 
-import java.nio.file.Path
+import scala.annotation.tailrec
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.{meta => m}
 
+import scala.meta._
 import scala.meta.internal.metals.MetalsEnrichments._
 import scala.meta.internal.metals.newScalaFile.NewFileTemplate
+import scala.meta.internal.parsing.Trees
 import scala.meta.internal.pc.Identifier
 import scala.meta.io.AbsolutePath
 
+import org.eclipse.lsp4j.Location
 import org.eclipse.lsp4j.Position
 import org.eclipse.lsp4j.Range
+import org.eclipse.lsp4j.ReferenceContext
+import org.eclipse.lsp4j.ReferenceParams
+import org.eclipse.lsp4j.TextDocumentIdentifier
 import org.eclipse.lsp4j.TextEdit
 import org.eclipse.lsp4j.WorkspaceEdit
 
-class PackageProvider(private val buildTargets: BuildTargets) {
+class PackageProvider(buildTargets: BuildTargets, trees: Trees) {
```

Comment:
```suggestion
class PackageProvider(buildTargets: BuildTargets, trees: Trees, refProvider: ReferenceProvider) {
```

Reply:
Fixed 
This required to change `val` to `var` in `packageProvider` declaration in `MetalsLanguageServer` and to shuffle initialization order a little